nfsn-cron.sh

```diff
@@ -17,8 +17,8 @@ if find certs -name cert.pem -type l \
 then
 	echo " + Certificte will expire in 30 days or less! SSH into this site and"
 	echo "   run the following commands to renew your certificates:"
-	echo "	cd ~/lets-nfsn.sh/letsencrypt.sh/"
-	echo "	./letsencrypt.sh --cron"
+	printf '	cd %q\n' "$PWD"
+	echo   '	./letsencrypt.sh --cron'
```

Comment:
Done. I wasn’t sure if letsencrypt.sh will do the right thing when it’s not run from it’s own directory.